controllers/resource_helper.go

```diff
@@ -77,20 +113,48 @@ func buildContainerSpec(instance *llamav1alpha1.LlamaStackDistribution, image st
 			MountPath: "/etc/llama-stack/",
 			ReadOnly:  true,
 		})
+	}
+
+	// Add CA bundle volume mount if TLS config is specified
+	if instance.Spec.Server.TLSConfig != nil && instance.Spec.Server.TLSConfig.CABundle != nil {
+		caBundleKey := instance.Spec.Server.TLSConfig.CABundle.ConfigMapKey
+		if caBundleKey == "" {
+			caBundleKey = "ca-bundle.crt"
+		}
+
+		container.VolumeMounts = append(container.VolumeMounts, corev1.VolumeMount{
+			Name:      "ca-bundle",
+			MountPath: "/etc/ssl/certs/" + caBundleKey,
```

Comment:
The path should be determined based on the Linux distro. Ideally, we would put this in `/etc/pki/ca-trust/source/anchors/` and then run `sudo update-ca-trust` so we don't even need to pass the cert later to the vLLM client. I might be stretching so we can revisit later, any thoughts?

Reply:
That would be more convenient, but I didn't find `/etc/pki/ca-trust/source/anchors/` available in the LlamaStackDistribution container (it's a Debian-based container image). So I went with a cross-platform middle ground solution.

Do you mean dynamically detect if `/etc/pki/ca-trust/source/anchors/` exists, and use it only if it is available?